tmva/tmva/src/TransformationHandler.cxx

```diff
@@ -173,6 +173,7 @@ const TMVA::Event* TMVA::TransformationHandler::InverseTransform( const Event* e
    // the inverse transformation
    TListIter trIt(&fTransformations, kIterBackward);
    std::vector< Int_t >::const_iterator rClsIt = fTransformationsReferenceClasses.end();
+   std::vector<Int_t>::const_iterator rClsBegin = fTransformationsReferenceClasses.begin();
```

Comment:
Instead of using .end(), would not be better to use begin() + size()-1 and starting element to iterate in reverse order ?

Reply:
If you want to change the code, it's fine with me. Here I just fix an assertion